### DISCUSSION TITLE
Cant clone a repostariy from git - Proxy Error - Solved

### DISCUSSION THREAD

**Discussion by OmriOv** [community/community#70725] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I can't seem to clone projects from Git to my local computer - it shows me a proxy error (image attached)
It also prevents me from pushing changes if i set my own repository,

Do you know where and how I can change it?

![image](https://github.com/community/community/assets/121634663/4de4143f-8fb1-4e2c-8df6-479f2a665a3b)

**Comment by itsdotnet** [comment]
Try other options
I mean if you cloning with HTTP link try SSH
Or opposite
 
Few days ago I got errors like this when I cloning project with SSH link
and I try HTTP link then it worked.

**Reply by OmriOv** [reply]
Tried cloning it with HTTP, SSH and locally 

Nothing seems to work, I still get the same proxy error message

**Reply by itsdotnet** [reply]
Which app is this you are trying to clone

**Reply by Anything-That-Works** [reply]
Try using the terminal to clone the project, maybe the issue is in GUI you are using

**Reply by itsdotnet** [reply]
Yes terminal is the best option when there is problem with version control

**Reply by OmriOv** [reply]
Same error in Git Bash
![image](https://github.com/community/community/assets/121634663/4e5c5305-a485-4159-8712-3ca3d1110004)

**Reply by Anything-That-Works** [reply]
Try the solution from here. 
https://stackoverflow.com/questions/27430239/git-could-not-resolve-proxy

**Reply by OmriOv** [reply]
Thanks! a comment from here helped me fixed it

**Comment by OmriOv** [comment, marked as answer]
Thank you! The issue is solved 👍 
I used this:

Solution to this problem is to remove the git proxies.

Open your terminal through the administrator.
Then hit command git config --global -l
Then unset all HTTP and https proxies by below command
git config --global --unset-all http.proxy & git config --global --unset-all https.proxy
Now you can check it is removed from the list or not with the same command git config --global -l
